game/src/main/kotlin/world/gregs/voidps/world/map/lumbridge/chicken_farm/HatchetLogs.kts

```diff
@@ -12,7 +12,7 @@ import java.util.concurrent.TimeUnit
 
 on<ObjectOption>({ obj.id == "hatchet_logs" && option == "Take-hatchet" }, Priority.HIGH) { player: Player ->
```

Comment:
```suggestion
on<ObjectOption>({ obj.id == "hatchet_logs" && option == "Take-hatchet" }) { player: Player ->
```

Idk why I put it as high here either, might as well change it at the same time

Reply:
i used this code in the trapdoors.kts that's why it was the same lol